src/js/tracing/reactnativetracing.ts

```diff
@@ -132,6 +145,17 @@ export class ReactNativeTracing implements Integration {
     this.options = {
       ...defaultReactNativeTracingOptions,
       ...options,
+      finalTimeoutMs: options.finalTimeoutMs
+        // eslint-disable-next-line deprecation/deprecation
+        ?? (typeof options.maxTransactionDuration === 'number'
```

Comment:
why do we need to check if its a `number`?

Reply:
because of the recalculation from s to ms.

Reply:
`maxTransactionDuration` always has a default value anyway, why would that be undefined?

Reply:
because here we are calculation with user input and that can be undefined.